micrometer-core/src/test/java/io/micrometer/core/instrument/binder/httpcomponents/hc5/MicrometerHttpRequestExecutorTest.java

```diff
@@ -68,19 +74,23 @@ void timeSuccessful(boolean configureObservationRegistry, @WiremockResolver.Wire
             throws IOException {
         server.stubFor(any(anyUrl()));
         CloseableHttpClient client = client(executor(false, configureObservationRegistry));
-        EntityUtils.consume(client.execute(new HttpGet(server.baseUrl())).getEntity());
+        execute(client, new HttpGet(server.baseUrl()));
         assertThat(registry.get(EXPECTED_METER_NAME).timer().count()).isEqualTo(1L);
     }
 
+    private void execute(CloseableHttpClient client, ClassicHttpRequest request) throws IOException {
+        EntityUtils.consume(client.execute(request, NOOP_RESPONSE_HANDLER).getEntity());
```

Comment:
`CloseableHttpClient.execute(ClassicHttpRequest)` has been deprecated on the current version, and its replacement requires an `HttpClientResponseHandler` instance. I didn't look into it in detail and simply used no-op version of it which seems sufficient.